nf_core/lint.py

```diff
@@ -429,16 +430,13 @@ def check_nextflow_config(self):
             ['trace.file'],
             ['report.file'],
             ['dag.file'],
-            ['params.reads','params.input','params.design'],
-            ['process.container'],
-            ['params.single_end']
+            ['process.container']
         ]
         # Old depreciated vars - fail if present
         config_fail_ifdefined = [
             'params.version',
             'params.nf_required_version',
             'params.container',
-            'params.singleEnd',
```

Comment:
Why are you removing this? I think it's still good to fail if we come across the old-style of the parameter, as it suggests that the template hasn't been updated properly. Even if we are no long warning about the absence of the new-style.

Reply:
Added back in: https://github.com/nf-core/tools/pull/556/commits/d1252c22b53eb380a5aee313f96de7d3834792d7